File: WindowsApplication1/WindowsApplication1/Casual1.vb
Public Class Casual1
    Private C1 As Casual = New Casual()

    Private Sub btn_Click(sender As Object, e As EventArgs) Handles btn.Click
        C1.Pmarca = txtMarca.Text
        C1.Ptalla = CInt(txtTalla.Text)
        C1.Pevento = txtEvento.Text
        txtMarca.Text = ""
        txtTalla.Text = ""
        txtEvento.Text = ""
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        MsgBox("Marca: " + C1.Pmarca + vbNewLine + "Talla: " + CStr(C1.Ptalla) + vbNewLine + "Evento: " + C1.Pevento)
    End Sub
End Class

File: HCSZapato/HCSZapato/Casual1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace HCSZapato
{
    public partial class Casual1 : Form
    {
        Casual C1 = new Casual();
        public Casual1()
        {
            InitializeComponent();
        }

        private void btn_Click(object sender, EventArgs e)
        {
            C1.setMarca(txtMarca.Text);
            C1.setTalla(Int32.Parse(txtTalla.Text));
            C1.setEvento(txtEvento.Text);
            txtMarca.Text = "";
            txtTalla.Text = "";
            txtEvento.Text = "";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            txtMarca.Text = C1.getMarca();
            txtTalla.Text = C1.getTalla().ToString();
            txtEvento.Text = C1.getEvento();
        }
    }
}
